client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -90,173 +104,182 @@ export const StockLineForm: FC<StockLineFormProps> = ({
     return () => document.body.removeEventListener('keydown', handleKeyDown);
   }, []);
 
+  if (loading) return null;
+
   return (
-    <Grid
-      display="flex"
-      flex={1}
-      container
-      paddingTop={2}
-      paddingBottom={1}
-      width="100%"
-      flexWrap="nowrap"
-    >
+    <DetailContainer>
       <Grid
-        container
         display="flex"
         flex={1}
-        flexBasis="50%"
-        flexDirection="column"
-        gap={1}
+        container
+        paddingTop={2}
+        paddingBottom={1}
+        width="100%"
+        flexWrap="nowrap"
+        maxWidth={900}
+        gap={footerProps ? 10 : undefined}
       >
-        <StyledInputRow
-          label={t('label.num-packs')}
-          Input={
-            <NumericTextInput
-              autoFocus
-              disabled={!packEditable}
-              width={160}
-              value={draft.totalNumberOfPacks}
-              onChange={totalNumberOfPacks => onUpdate({ totalNumberOfPacks })}
-            />
-          }
-        />
-        <StyledInputRow
```

Comment:
This stuff hasn't really changed, probably want to disable white-space changes when reviewing this.